app/views/vacancies/_filters.html.haml

```diff
@@ -24,6 +24,11 @@
     .form-group
       = label_tag 'phase', t('jobs.filters.education_phase'), class: 'form-label'
       = select_tag 'phase', options_for_select(school_phase_options, phase), class: 'form-control form-control-block', include_blank: 'Any'
+    .form-group
+      %span.form-label= t('jobs.filters.newly_qualified_teacher')
```

Comment:
I spoke to UR who confirmed that they don't want this label.
Sorry about the lack of clarity on this on the trello card.

Reply:
Ok done